org.spesmilo.Electrum.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
The AppData file lacks some data, such as `content_rating` tag (OARS) or `launchable` tag.
https://github.com/flathub/flathub/wiki/AppData-Guidelines
https://hughsie.github.io/oars/generate.html